plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveSessionProperties.java

```diff
@@ -47,6 +47,7 @@
 {
     private static final String BUCKET_EXECUTION_ENABLED = "bucket_execution_enabled";
     private static final String VALIDATE_BUCKETING = "validate_bucketing";
+    private static final String PARALLEL_PARTITIONED_BUCKETED_INSERT = "use_partitioned_bucket_function";
```

Comment:
why not `use_parallel_partitioned_bucketed_inserts` or `parallel_partitioned_bucketed_inserts_enabled`.
I see no reason to have different wording for config and session property.